test/src/test/resources/fixtures/hydration/basic-hydration-with-default-array-argument-values.yml

```diff
@@ -0,0 +1,114 @@
+name: "basic hydration with default array argument values"
+enabled: true
+overallSchema:
+  # language=GraphQL
+  service2: |
+    type Query {
+      barById(id: ID, test: [String]): Bar
+    }
+    type Bar {
+      id: ID
+      name: String
+    }
+  # language=GraphQL
+  service1: |
+    type Query {
+      foo: Foo
+    }
+    type Foo {
+      id: ID
+      bar(test: [String] = ["Hello", "World"]): Bar @hydrated(
```

Comment:
Hydration is here. It uses `$argument.test` with the default value of `["Hello", "World"]`